live/src/agama-installer.kiwi

```diff
@@ -150,6 +150,13 @@
         <package name="xorg-x11-server"/>
         <package name="xorg-x11-fonts-core"/>
         <package name="xf86-input-libinput" />
+        <package name="xf86-input-wacom" />
```

Comment:
wacom was taken from installation-images

Reply:
note wacom is used on all archs except s390